frontend/amundsen_application/proxy/issue_tracker_clients/jira_client.py

```diff
@@ -206,7 +233,10 @@ def _sort_issues(self, issues: List[Issue]) -> List[DataIssue]:
             if not issue.fields.resolution:
                 open.append(data_issue)
             else:
-                closed.append(data_issue)
+                created_date = datetime.strptime(issue.fields.created.split(".")[0], '%Y-%m-%dT%H:%M:%S')
```

Comment:
I see why you had to resort to this given the jira client and api ugliness. My suggestion though is instead to put this as a condition in the JQL. That seems less brittle. i think the relevant bit of JQL is `AND updated > -30d `. Note I'm also advising to switch to `updated` instead of `created`.

Reply:
If you switch to `updated` then 90 is probably too long. Also should the number of days be a config or is that overkill?

Reply:
Changed to use `updated` and 30 days instead of 90 based on your suggestions - as for using a config, I decided it seemed a bit unnecessary since this is only for the ones that are displayed (which hopefully people don't care to see ones that old anyway), and anyone will still be able to access all closed ones from the new `View X closed issues` link, but lmk if you think otherwise